src/Uno.SourceGeneration.Engine.Shared/Workspace/ProjectInfoBuilder.cs

```diff
@@ -80,7 +79,16 @@ public static ProjectInfo CreateProjectInfo(ProjectFileInfo projectFileInfo)
 				analyzerReferences: Enumerable.Empty<AnalyzerReference>(),
 				additionalDocuments: additionalDocuments,
 				isSubmission: false,
-				hostObjectType: null);
+				hostObjectType: null).WithAnalyzerConfigDocuments(commandLineArgs.AnalyzerConfigPaths.Select(CreateDocumentInfo));
+
+			DocumentInfo CreateDocumentInfo(string path)
+			{
+				return DocumentInfo.Create(
+					DocumentId.CreateNewId(projectId, path),
+					name: path,
+					filePath: path,
+					loader: new FileTextLoader(path, commandLineArgs.Encoding));
+			}
```

Comment:
The `ProjectInfo` we created never had analyzer config documents, and `commandLineArgs` didn't have it, unfortunately. For some reason, I was getting into this condition when testing locally:

https://github.com/unoplatform/Uno.SourceGeneration/blob/d859a37defd43065b215a0d2dd3f68f6821e5064/src/Uno.SourceGeneration.Engine.Shared/Workspace/ProjectFileInfo.cs#L299

The "best effort" code path didn't add analyzerconfig command-line arguments. So I added them in `ReadAnalyzerConfig`